adhocracy4/modules/views.py

```diff
@@ -0,0 +1,45 @@
+from django.core.urlresolvers import reverse
+from django.shortcuts import get_object_or_404
+from django.views import generic
+
+from adhocracy4.rules import mixins as rules_mixins
+
+from . import models
+
+
+class ModuleDetailView(rules_mixins.PermissionRequiredMixin,
+                       generic.DetailView):
+    model = models.Module
+    permission_required = 'a4projects.view_project'
+    slug_url_kwarg = 'module_slug'
+
+    def dispatch(self, request, *args, **kwargs):
+        module = self.get_object()
+        if module.is_active_in_project:
+            return ModuleRedirectView.as_view()(request, *args, **kwargs)
```

Comment:
why did you use the RedirectView? I guess it's a leftover from a previous version. but we could just return directly via
```
from django.http.response import HttpResponseRedirect
...
    if module.is_active_in_project:
        url = reverse('project-detail', kwargs={'slug': module.project.slug})
        return HttpResponseRedirect(url)
```